src/layouts/styles.scss

```diff
@@ -1,3 +1,4 @@
+@import '../stylesheets/testimonial.scss';
```

Comment:
Add custom styles after font loading

Reply:
Even better, import `testimonial.scss` directly in Testimonial component

Reply:
Alright